src/peps_util.jl

```diff
@@ -163,7 +163,9 @@ struct Operator
     dir::Op_Type
 end
 
-getDirectional(ops::Vector{Operator}, dir::Op_Type) = collect(filter(x->x.dir==dir, ops))
+function getDirectional(ops::Vector{Operator}, dir::Op_Type) 
```

Comment:
Really don't think this needs to be a function rather than a one-liner

Reply:
No problem, changed it back